service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -754,4 +754,26 @@ public Map<String, Object> addRelatingLA(String migrationId, Long caseId) {
 
         return Map.of("relatingLA", relatingLA.get());
     }
+
+
+    public Map<String, Object> removeSealedCMO(CaseData caseData,
+                                               String migrationId,
+                                               UUID expectedCMOId) {
+
+        List<Element<HearingOrder>> updatedSealedCMOs = ElementUtils.removeElementWithUUID(
+            caseData.getSealedCMOs(), expectedCMOId);
+
+        if (updatedSealedCMOs.size() != caseData.getSealedCMOs().size() - 1) {
+            throw new AssertionError(format(
+                "Migration {id = %s, case reference = %s}, Sealed CMO not found, %s",
+                migrationId, caseData.getId(), expectedCMOId));
+        }
+
+        List<Element<HearingOrder>> updatedOrdersToBeSent = ElementUtils.removeElementWithUUID(
```

Comment:
Do we need a similar assertion on this removal as you have on the removal of sealed CMO above?

Reply:
yes, it is in the unit test. already.
assertThat(updatedFields).extracting("ordersToBeSent").isEqualTo(expectedList);
assertThat(updatedFields).extracting("ordersToBeSent").isEqualTo(List.of());

Reply:
updatedOrdersToBeSent exist in this ticket, but not a must for other case. But yeah, maybe worth adding it, and adjust it if other future ticket don't need it